mrp_unbuild_cust_alu_analytics/models/mrp_unbuild_incidence.py

```diff
@@ -32,6 +32,8 @@ class MrpUnbuildIncidence(models.Model):
         string='Description'
     )
 
+    start_incidence = fields.Datetime(string='Start Incidence')
+
```

Comment:
Better (TOC)
```suggestion
    start_incidence = fields.Datetime(string='Start Incidence')

```

Reply:
Not agree... I should name field and label as `incidence_start` and "Incidence Start", in proper English IMO... And for this case (or the current one), string attribute is not required, because Odoo will put the desired one without indicating it, at least in this case

Reply:
Anyway, please don't change anything now, this is a lesson for the next development